src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/Ordering/OrderingHelper.cs

```diff
@@ -453,27 +335,30 @@ private static bool TryMoveElements(ReadOnlyCollection<ProjectItemElement> eleme
             }
 
             // Sanity check
-            var atLeastOneElementMoved = false;
+            var didAllElementsMove = true;
```

Comment:
Why the change in meaning here?  Can you document the semantics of the return value in the doc comment?

Reply:
I wanted it to be for all elements moved. It helps ensure the tests absolutely are correct. Yes, I can add a doc comment for it.